backend/dataall/modules/dataset_sharing/services/share_managers/s3_access_point_share_manager.py

```diff
@@ -127,10 +127,54 @@ def manage_bucket_policy(self):
         logger.info(f"Manage Bucket policy for {self.bucket_name}")
 
         s3_client = S3Client(self.source_account_id, self.source_environment.region)
```

Comment:
Minor: this client is never used, it is duplicated in line 154

Reply:
good catch - removed